Backoffice/Tests/EventSubscriber/BlockTransverseCacheSubscriberTest.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+namespace OpenOrchestra\Backoffice\Tests\EventSubscriber;
+
+use OpenOrchestra\Backoffice\EventSubscriber\BlockTransverseCacheSubscriber;
+use OpenOrchestra\BaseBundle\Tests\AbstractTest\AbstractBaseTestCase;
+use OpenOrchestra\ModelInterface\Model\NodeInterface;
+use Phake;
+use OpenOrchestra\ModelInterface\NodeEvents;
+
+/**
+ * Class BlockTransverseCacheSubscriberTest
+ */
+class BlockTransverseCacheSubscriberTest extends AbstractBaseTestCase
```

Comment:
@alavieille : ``TransverseBlockCacheSubscriberTest`` instead of ``BlockTransverseCacheSubscriberTest``